### PR TITLE
Re-config email filter 

### PR DESCRIPTION
**Pull request by Sm00thiee**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -23,11 +23,14 @@
...
-GOOGLE_CLIENT_SECRET = '7TMNNst1I5ueVjacoQDa1sJg'
-GOOGLE_DISCOVERY_URL = (
-    "https://accounts.google.com/.well-known/openid-configuration"
-)
+import json
+data = json.load(open('app_key.json'))
+client_key = data['google_login'][0]['client_key']
+client_secret = data['google_login'][0]['client_secret']
+discovery_url = data['google_login'][0]['discovery_url']
+GOOGLE_CLIENT_ID = client_key
```
Comment (lthoangg) [inline app.py:33]: Don't need to create 3 new variables to reference 3 others.

File: app.py [hunk 2]
```diff
@@ -155,44 +159,36 @@ def callback():
...
+            id_ = user.get_id()
+            name = user.getName()
+            # temp = name.split()
+            # name = temp[1] + ' ' + temp[2] + ' ' + temp[0]
+            email = user.getEmail()
+            profile_pic = user.getprofile_pic()
+            mongoUsr.register(id_, name, email, profile_pic)
+            generate_password()
+            # Doesn't exist? Add to database
+            if not user.get(unique_id):
```
Comment (lthoangg) [inline app.py:180]: We should check if the account is created bf get their info btw. Put these "if statement" to line 168. And put other thing from 168 to 175 to "else statement".

**Review by lthoangg — COMMENTED** [review]
Several bugs need to be fixed.

**Comment by Sm00thiee** [comment]
Thanks myself for merging ;)

**Pull request merged**